history/characters/340000_night_elf.txt

```diff
@@ -147,14 +147,14 @@
 	disallow_random_traits = yes
 	20.7.15={ birth=yes trait=creature_night_elf effect = { make_important_lore_character_effect = yes } }
 	40.1.1={
-		# trait=class_warrior_7
```

Comment:
It will be more correct to make Illidan half-mage before he turned to Fel.

Reply:
Done?